### PR TITLE
Bold that save states save the RNG seed of games

### PR DESCRIPTION
**Pull request by Efadd**

### CODE REVIEW

**Comment by FearlessTobi** [comment]
Please squash.

File: src/responses/yuzu.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
...
       "scam": { "reply": "If you believe that a user is attempting to scam you or others, please report them to discord trust and safety and promptly block them. \nTo report the user, please follow this link <https://dis.gd/request>. \nFor more information about how to report them, please look here: <https://support.discord.com/hc/en-us/articles/360000291932-How-to-Properly-Report-Issues-to-Trust-Safety>. \nFinally, if this is the only server you share, please report the user (including screenshots of the conversation) to our moderation team, so that we can address the issue."},
+      "piracy": { "reply": "*yuzu* is legal, we don't support illegal activities. Dumping your purchased games and system files from your Switch is legal, downloading them is not.\n\nPlease reference the __Quickstart Guide__ in order to dump your games, keys, and system files from your Switch for use with yuzu. <https://yuzu-emu.org/help/quickstart/>\n\nAlso, please refresh yourself on the <#418589661783392267> you agreed to when you joined the server."},
```
Comment (FearlessTobi) [inline src/responses/yuzu.json:28]: unrelated to this pr

File: src/responses/citra.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
+      "states": {"reply": "Citra has two kinds of saves available to users, in-game saves and save states.\n\nIn-game saves are the ones you know from the 3DS. These can be shared between devices and are recommended for general use, as they do not corrupt as easily.\nSave states, on the other hand, save Citra's entire emulation state. This includes Citra's configuration, **a game's RNG seed**, as well as any faults in the emulation at the time of saving. This makes them very useful as a tool for practicing in-game tricks, or for usage in shiny hunting and other RNG related trigger events. However they're not suitable as replacements for in-game save files as the faults in the emulation state would keep building up until the emulation state is too unstable to load properly. They're also ***non-transferable*** because they are made with one setup and only one."},
```
Comment (FearlessTobi) [inline src/responses/citra.json:25]: why exactly is this needed?

**Review by FearlessTobi — CHANGES_REQUESTED** [review]

File: src/responses/yuzu.json (hunk 1 above)
Comment (Efadd) [reply src/responses/yuzu.json:28]: Need to trim commits on my end.

**Comment by Efadd** [comment]
I'll close, squash, and revert the Yuzu file change

**Pull request closed without merging**